File: bubbleSort.py
def bubbleSort(arr):
    n = len(arr)
    
    for i in range(n):
        
        for j in range(0, n-i-1):
            if arr[j] > arr[j+1] :
                arr[j], arr[j+1] = arr[j+1], arr[j]

arr = list()
filename = 'numbers.txt'
with open(filename) as fin:
    for line in fin:
        arr.append(line)
print(arr)  

bubbleSort(arr)

print ("Sorted array is:") 
for i in range(len(arr)): 
    print ("%s" %arr[i])

File: bubbleSort.R

bubblesort <- function(arr){
  
  n = length(arr)
  v = arr
  
  for(j in 1:(n-1))
  {
    for(i in 1:(n-j))
    {
      if(v[i+1]<v[i])
      {
        t = v[i+1]
        v[i+1] = v[i]
        v[i] = t    
      }
    }
  }
  print(v)
  arr = v 
}

y<-scan(file = "numbers.txt", what = numeric(), sep = "\n") 

sortedarray<-bubblesort(y)

print("Sorted array is:")

sortedarray
